docs/04-tables.md

```diff
@@ -2,8 +2,8 @@ Tables
 ======
 
 A Sleeper instance contains one or more tables. Each table has four important
-properties: a name, and a schema for storing data for that
-table, and its own state store for storing metadata about the table. 
+properties: a name, a schema for storing data for that table, a state store for 
+storing metadata about the table, and a flag to denote whether the table is online or not. 
```

Comment:
Can we either explain briefly what it means that a table is online or not, or link to more information in another document?